docs/HowToFAQ/ERROR_DuplicateImportName.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Duplicate name of import: ..."
+---
+
+This error results from importing two modules with the same name. For example
+```
+import A.Util
+import B.util
+```dafny
```

Comment:
Can you place the `dafny` tag on the first triple tick? And it seems to be missing on the fixed example.

Reply:
Fixed